templates/example_hello_world_ts/tsconfig.eslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "./tsconfig.json",
```

Comment:
i guess we can use the root tsconfig directly, or not? the dedicated eslint tsconfig is usually there to allow checking test files too, here it does not add any benefits

Reply:
Yes, it is redundant to specify it. I have Improved the .eslintr file to use minimal working config.